extensions/GitExtension/FileExplorerGitIntegration.UnitTest/WslIntegratorUnitTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using FileExplorerGitIntegration.Models;
+
+namespace FileExplorerGitIntegration.UnitTest;
+
+[TestClass]
+public class WslIntegratorUnitTests
+{
+    [TestMethod]
+    [DataRow("\\wsl$\\Ubuntu-20.04\\home\\user\\repo")]
+    [DataRow("\\wsl.localhost\\Ubuntu-20.04\\home\\user\\repo")]
+    [DataRow("\\wsl$\\Ubuntu\\home\\user\\repo")]
+    [DataRow("\\wsl.localhost\\Ubuntu\\home\\user\\repo")]
+    [DataRow("\\wsl.localhost\\Debian\\home\\user\\repo")]
+    [DataRow("\\wsl$\\kali-linux\\home\\user\\repo")]
+    [DataRow("\\wsl$\\Ubuntu-18.04\\home\\user\\testRepo")]
+    [DataRow("\\wsl.localhost\\Ubuntu-18.04\\home\\user\\testRepo")]
+    public void IsWSLRepoPositiveTests(string repositoryPath)
+    {
+        Assert.IsTrue(WslIntegrator.IsWSLRepo(repositoryPath));
+    }
+
+    [TestMethod]
+    [DataRow("C:\\Users\\foo\\bar")]
```

Comment:
Test that we reject wrong direction slashes.

Reply:
Made test modifications in latest changes. Thanks!